.github/ISSUE_TEMPLATE/suggest_wallet.md

```diff
@@ -6,98 +6,192 @@ labels: "wallet :purse:,"
+
+**What social links are there for the project?**
+
+<!-- Please provide social links for the wallet (Discord, Twitter, etc.) -->
+
+**What is your wallet's onboarding experience?**
+
+<!-- Please provide an explanation of where to find the user onboarding experience (links, is it built into the app, etc.), and what the onboarding experience is. -->
+
+### Device
+
+**Does the wallet have a mobile app? If yes, which operating systems are supported (iOS, Android)?**
+
+<!-- If the wallet has a mobile interface, please provide information and links to the app store for the operating systems supported. -->
+
+**Does the wallet have a desktop app? If yes, which operating systems are supported (Windows, Mac, Linux)?**
+
+<!-- If the wallet has a desktop interface, please provide links and information for the operating systems supported. -->
+
+**Does the wallet have a browser extension? If yes, which browsers are supported (Chromium, Firefox, Safari)?**
 
-<!-- Do users have access to their public and/or private keys? If your company/project were to disappear, would users still be able to access their funds? -->
+<!-- If the wallet has a browser extension, please provide links and information for the browsers that are supported. -->
+
+**Is it a hardware wallet?**
+
+<!-- How is it used / how does it connect (e.g. USB)? What software wallets does it integrate with (e.g. MetaMask)? -->
+
+### Security
+
+**Is the wallet code open-sourced?**
+
+<!-- If yes, please provide a direct link to the repository. -->
+
+**What license is the wallet software released under?**
+
+<!-- Please provide information on the software license used for the wallet. -->
+
+**Is your wallet custodial, or non-custodial?**
+
+<!-- Do users have access to their public and private keys? Does anyone else? If your company/project were to disappear, would users still be able to access their funds? -->
 
 **Please describe the measures taken to ensure the wallet's security and provide documentation wherever possible**
 
-<!-- Please provide a link to a report or repo. If you haven't been audited but think your wallet should be listed anyway, explain here. -->
+<!-- Please provide a link to any security audits or code reports. If you haven't been audited but think your wallet should be listed anyway, explain here. -->
 
-**Does the wallet support layer 2 networks?**
+**Spam protection?**
 
-<!-- Please list which L2 solutions the wallet supports. -->
+<!-- Does the wallet employ any practices to warn users against potential spam (e.g. when interacting with suspicious accounts/contracts)? -->
 
-**Can the wallet be used with arbitrary Ethereum RPC endpoint?**
+### Features
 
-<!-- Does wallet allow user to use the wallet with own Eth client? -->
+**Does the wallet have hardware wallet support?**
 
-**Does the wallet have fiat on-ramps?**
+<!-- Please provide information on how a user can connect a hardware wallet to this wallet. -->
```

Comment:
Curious how we would use this info? Wouldn't this one be more of a binary yes/no?

Reply:
Yeah, this is more binary. My idea here is a link to documentation like this: https://docs.metamask.io/guide/signing-data.html#signing-data-with-metamask